commit 45119785c7efbaa1defc3fffd05cb41e39992271
Author: Cole Robinson <crobinso@redhat.com>
Date:   Thu Apr 5 13:28:39 2012 -0400

    schemas: Fix key for default graphics type
    
    What we installed didn't match what the app was checking and setting.
    Have it match the app key so we don't overwrite users who have changed the
    default via the prefs dialog.

diff --git a/src/virt-manager.schemas.in b/src/virt-manager.schemas.in
index dfcd7ba..8154534 100644
--- a/src/virt-manager.schemas.in
+++ b/src/virt-manager.schemas.in
@@ -196,8 +196,8 @@
     </schema>
 
     <schema>
-      <key>/schemas/apps/::PACKAGE::/new-vm/graphics-type</key>
-      <applyto>/apps/::PACKAGE::/new-vm/graphics-type</applyto>
+      <key>/schemas/apps/::PACKAGE::/new-vm/graphics_type</key>
+      <applyto>/apps/::PACKAGE::/new-vm/graphics_type</applyto>
       <owner>::PACKAGE::</owner>
       <type>string</type>
       <default>::DEFAULT_GRAPHICS::</default>
